tendrl/node_agent/node_sync/disk_sync.py

```diff
@@ -231,6 +234,9 @@ def get_disk_details():
                     if "by-id/virtio" in entry:
                         devlist['disk_id'] = entry.split('/')[-1]
                         break
+            elif "VMware" in devlist["vendor"]:
+                 devlist["disk_id"] = (devlist["vendor"] + "_" +
+                    devlist["device"] + "_" + devlist["hardware_id"])
```

Comment:
```suggestion
                 devlist["disk_id"] = "{vendor}_{device}_{hardware_id}".format(**devlist)
```
makes it much efficient than the `+` concatenation.